.github/workflows/ci.yml

```diff
@@ -12,14 +12,18 @@ jobs:
     strategy:
       matrix:
         otp: ['22.3', '21.3']
```

Comment:
Maybe we could use OTP 23? Not required now of course.

Reply:
right now we cannot switch to OTP. [pg2](http://erlang.org/doc/man/pg2.html) is deprecated in OTP23 and xref check is failing, the rest checks are passing well w/o any issues. 

So we need to first switch from [pg2](http://erlang.org/doc/man/pg2.html) to something else, but if we switch to [pg](http://erlang.org/doc/man/pg.html) as OTP documentation suggests we limit ourselves to OTP23 and higher version since it's introduced only in OTP23.